Multiple field order by

posva opened the pull request:
This is refers to the feature request #2518

I replaced the deconstructing array with a simple array because it makes it easier to use an array from data or computed properties: `files | orderBy sortKeys` instead of `files | orderBy 'name' 'size'` with `sortKeys = ['name', 'size']`. Tell me your thoughts about it

posva commented:
Ping @yyx990803

yyx990803 commented on src/filters/array-filters.js:
```diff
@@ -93,6 +97,16 @@ export function orderBy (arr, sortKey, reverse) {
+
+  // sort on a copy to avoid mutating original array
+  return arr.slice().sort(function (a, b) {
+    return recursiveCompare(a, b, 0)
```
we can jsut `sort(recursiveCompare)` if we do `i = i || 0` in it right?

posva commented:
@yyx990803 Thanks for the update, I didn't think about the unnecessary function call :smile: . I fixed that and also squashed everything into a single commit. I made a backup in case you prefer it the other way, just tell me if you want me to switch back to the 4 commits version :wink:

yyx990803 commented:
:+1: good job!
